Engine/HistoricalData/SubscriptionDataReaderHistoryProvider.cs

```diff
@@ -128,7 +134,7 @@ private Subscription CreateSubscription(HistoryRequest request, DateTime start,
                 _factorFileProvider,
                 dataReader,
                 mapFileResolver,
-                false);
+                true);
```

Comment:
Seems to me the only reason the `includeAuxiliaryData` parameter was added was for this case, not clear to me why?. Maybe we don't need it anymore?
Some history: https://github.com/QuantConnect/Lean/commit/4ae1f4f5b61e1ac5445dc6e9f84cdd4512a151a7#diff-fff27851162988132325d72186ba10a3R103 and https://github.com/QuantConnect/Lean/commit/062eac829fd8884676145004c7ecc3e59e328839